docs/01/b.md

```diff
@@ -2,13 +2,13 @@
   [algebra]: https://github.com/non/algebra
   [Equivalence]: http://en.wikipedia.org/wiki/Equivalence_relation
 
-## Eq
+## `Eq`
```

Comment:
You previously had backticks on fourth level headers in some places. Do you want me to make that consistent or leave it alone? Here're the real questions: do you want backticks on any headers? If so, from what level on?